src/js/facility-locator/components/ServicesAtFacility.jsx

```diff
@@ -131,15 +129,16 @@ class ServicesAtFacility extends Component {
     // TODO: clean up once we have real data
     return (
       <div>
-        {facility.type === 'va_health_facility' ? this.renderHealthServices() : null}
+        <p style={{ margin: '0 0 0.5em' }}>Services current as of <strong>{moment().format('MMMM D, YYYY')}</strong></p>
```

Comment:
I'm not sure where we want to pull the date from. It's showing the current day right now but we can change it if there is a canonical source for when service data was last updated